src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        String randomColor = colorSupplier.getRandomColor().toString();
+        int randomNumber = random.nextInt(101);
```

Comment:
Magic number '101' should be declared as a constant to make the code more readable and maintainable. For example, private static final int MAX_NUMBER = 101; and then use this constant in the nextInt() call.